admin/bump_version.py

```diff
@@ -124,13 +129,30 @@ def updateCopyrightString(filename, lines):
             newLines.append(line)
             break
     else:
-        raise RuntimeError(f"End of copyright not found in {filename}.")
+        raise RuntimeError(f"End of copyright not found in {filename!r}.")
 
     newLines.extend(iterLines)
 
     return newLines
 
 
+def updateCallOfReleaseChecklist(filename: str, lines: List[str]):
+    newLines = []
+
+    callReleaseChecklist = "admin/release_checklist"
+    callFound = False
+    iterLines = iter(lines)
+    for line in iterLines:
```

Comment:
Explicit iter() call seems unnecessary here since we don't want to suspend/resume or manually advance it here.

```suggestion
    for line in lines:
```